README.md

```diff
@@ -338,11 +340,13 @@ Supports an on-premise [GitLab](http://gitlab.com) Community Edition/Enterprise
 
 | Setting            | Description
 |--------------------|-------------------------------------------------------------------------------------------------------------
-| `url`              | GitLab server http(s) address string
-| `token`            | Secret token string for the existing user to be used to authenticate against GitLab REST API
-| `slugs`            | List of project slugs to display and check for builds. Defaults to `*/*` for all projects you have access to. Use `/*` when specifying group slug to include projects only from current group and `/**` to also include subgroups. Optional 'ref' attribute can be used to specify the branch.
+| `debug`            | Print all available logs to console.
+| `url`              | GitLab server http(s) address string.
+| `token`            | Secret token string for the existing user to be used to authenticate against GitLab REST API.
+| `slugs`            | List of project slugs to display and check for builds. Defaults to `*/*` for all projects you have .access to. Use `/*` when specifying group slug to include projects only from current group and `/**` to also include subgroups. Optional 'ref' attribute can be used to specify the branch.
 | `intervals`        | How often (in integer of milliseconds) ...
 | `additional_query` | Add [additional query parameters](https://gitlab.com/help/api/projects.md) so not too many projects are fetched.
+| `main_group`       | Gitlab needs a main group to analyze the project only you want to monitor, otherwise, you will get the whole world.
```

Comment:
What happens, if this property is not set? Will existing GitLab configurations crash after an upgrade of the build monitor?